pkgs/applications/editors/emacs-modes/telega/libtgvoip.nix

```diff
@@ -0,0 +1,34 @@
+{
+pkgs
+, clangStdenv
+, libopus
+, openssl
+, libpulseaudio
+, alsaLib
+, pkgconfig
+, writeText
+, llvmPackages
+, libtool
+, writeShellScriptBin
+}:
+clangStdenv.mkDerivation {
```

Comment:
Is `clangStdenv` required or could we use normal `stdenv`?

Reply:
libtool was whining about unspecified --tag, and whole project is a mix of C and C++. So I just switched to clang and it built it without a problem.